server/Episememe.Application.Tests/FeatureTests/RemoveFavoriteMedia/WhenRemovingMediaFromFavorites.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Episememe.Application.Features.RemoveFavoriteMedia;
+using Episememe.Application.Interfaces;
+using Episememe.Application.Tests.Helpers;
+using Episememe.Domain.Entities;
+using FluentAssertions;
+using System;
+using System.Data.Common;
+using System.Threading;
+using Xunit;
+
+namespace Episememe.Application.Tests.FeatureTests.RemoveFavoriteMedia
+{
+    public class WhenRemovingMediaFromFavorites : IDisposable
```

Comment:
Add tests for idempotency, e.g. create a test when handler is ran 3 times, and then check if results are ok